.github/workflows/docker-publish.yml

```diff
@@ -68,7 +68,7 @@ jobs:
         if: ${{ github.event_name != 'pull_request' }}
         uses: actions/attest-build-provenance@v1
         with:
-          subject-name: index.docker.io/${{ env.VERSION }}
+          subject-name: index.docker.io/rustvmm/dev
```

Comment:
What about add a new command to `docker.sh` and set another env variable in the `Generate next docker tag` step?

Something like this:
```
REGISTRY=$(./docker.sh print-registry)
echo "REGISTRY=${REGISTRY}" >> $GITHUB_ENV
```

Reply:
Or just:
```
echo "REGISTRY=$(./docker.sh print-registry)" >> $GITHUB_ENV
```

Reply:
s/REGISTRY/CONTAINER (or something like that, I was confused, the registry is index.docker.io)